images/macos/provision/core/python.sh

```diff
@@ -5,14 +5,15 @@ echo "Installing Python Tooling"
 
 echo "Brew Installing Python 3"
 # Workaround to have both 3.8 & 3.9(which required by some brew formulas) in the system, but only 3.8 is linked
-/usr/local/bin/brew install python@3.8
-/usr/local/bin/brew install python@3.9
-/usr/local/bin/brew unlink python@3.9
-/usr/local/bin/brew unlink python@3.8
-/usr/local/bin/brew link python@3.8
+brew install python@3.8
+brew install python@3.9
+brew unlink python@3.9
+brew unlink python@3.8
+brew link python@3.8
```

Comment:
I thought we are removing this workaround here https://github.com/actions/virtual-environments/pull/1867/files, maybe makes sense to combine PRs?

Reply:
I am not ready to merge that PR yet. I am waiting for Python tests to be ready.